dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDetailsIvyDevWfBean.java

```diff
@@ -158,6 +159,52 @@ private boolean canResume() {
     return isActivator || isWorker;
   }
 
+  public boolean showInfoBanner() {
+    boolean showBanner = selectedTask.getActivator().isMember(ISession.current()) && List.of(
```

Comment:
Instead of getActivator() which can return null use method activator() which always returns something (No NPE).